### PR TITLE
add blog post for March 2021 meetup

### PR DESCRIPTION
**Pull request by rahul-kumi**

### CODE REVIEW

**Comment by rahul-kumi** [comment]
@anistark, @abhiramr
Please do share your reviews.

**Review by abhiramr — APPROVED** [review]
Very cool 😎

File: _posts/2021-03-20-march-meetup.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	- Image to image translations with [pipx2pix model](https://phillipi.github.io/pix2pix/).
+	- Google's [quick draw](https://quickdraw.withgoogle.com/).
+	- Interactive sketch prediction.
+- Application of NLP through the use of GPT-2 APIs
+
+The demo packed talk session was followed by 2 lightning talks. The first of which was presented by Bangpyper's veteran member [Kracekumar](https://kracekumar.com/). He spoke about a tool that he built over a weekend to render Jupyter Notebook content on terminals called JUT. The need for this solution was stated as the inconvenience of always having to rely on browsers and editors to view them. The approach that was taken here to render notebooks on terminals was to use the JSON source of .ipynb files to parse using the available schema and display them on the terminal. Karce took us through the different technical hurdles that he had in building this solution and also listed the features he has planned to integrate. JUT is open for collaborations and suggestions, find the project here: [https://github.com/kracekumar/jut](https://github.com/kracekumar/jut)
```
Comment (kracekumar) [inline _posts/2021-03-20-march-meetup.md:38]: Typo in my name "karce took us..."

Comment (rahul-kumi) [reply _posts/2021-03-20-march-meetup.md:38]: Oops, my apologies @kracekumar...I have fixed this in PR #47.
Thanks for pointing this out.

**Pull request merged**